db-service/test/cqn2sql/__snapshots__/expression.test.js.snap

```diff
@@ -55,7 +55,7 @@ exports[`expressions with complex xpr 1`] = `
 
 exports[`expressions with exists 1`] = `
 {
-  "sql": "SELECT Foo.ID,Foo.a,Foo.b,Foo.c,Foo.x FROM Foo as Foo WHERE exists (SELECT Foo2.name FROM Foo2 as Foo2) or not exists (SELECT Foo2.name FROM Foo2 as Foo2)",
+  "sql": "SELECT Foo.ID,Foo.a,Foo.b,Foo.c,Foo.x FROM Foo as Foo WHERE exists (SELECT Foo2.name FROM Foo2 as Foo2) or not exists (SELECT Foo22.name FROM Foo2 as Foo22)",
```

Comment:
this `Foo22` looked wrong at a first glimpse. However, as we `SELECT FROM Foo2` the 2nd `Foo2` alias becomes `Foo22`, so all good :)